conf/messages

```diff
@@ -454,12 +460,15 @@ error.propertyRepresentation.agentAlreadyAppointed=This agent has already been a
 
 #Show your agents
 propertyRepresentation.showAgents.title=Your agents
+propertyRepresentation.agentProperties.assigned=Assigned properties
+propertyRepresentation.manage.agent=Manage this agent
 propertyRepresentation.showAgents.properties={0} of {1} properties
 propertyRepresentation.showAgents.viewAgentDProperties=<a class="govuk-link" href="{0}">View</a>
+error.propertyRepresentation.agentProperties.filter=Search Properties - you must enter something to search for
+error.agentProperties.filter=You must enter either address or Local authority reference something to search for
```

Comment:
Maybe check the text. The second one is not acceptable english. I'm not sure about the first. Better check.

Reply:
I have edited, I have changed to same as one in Your Properties screen now and I will check with Karren about it When Search Functionality applies